examples/3d/load_gltf.rs

```diff
@@ -21,3 +22,36 @@ fn setup(commands: &mut Commands, asset_server: Res<AssetServer>) {
             ..Default::default()
         });
 }
+
+// When pressing spacebar, it will display only part of the model, rotating between the parts
+fn display_only_parts(
```

Comment:
While I like that this shows how to select based on name and _running_ the app is fun, I'm a little hesitant to include it:

1. Its a slightly roundabout way to "hide" things. Im assuming we can't use Visible::is_visible because the named nodes are "parents" that we don't actually draw. But toggling visibility would be a bit more self-documenting / idiomatic. As-is its hard to say that this is how people should solve the "hide by name" problem generically.
2. It makes the example less minimal. In general I want each example to be a small and clear illustration of a single feature.

Reply:
yeah I agree but it was fun to play with... that's why I did it in a different commit, to be extra easy to remove...

Reply:
thanks :)